config/routes.rb

```diff
@@ -38,6 +38,13 @@
     end
   end
 
+  resources :collections do
+    member do
+      get :manifest, defaults: { format: :json }, as: "manifest_curation_concerns"
+      get :manifest, defaults: { format: :json }
```

Comment:
Can you pass multiple things to `as`?  I think it's fine to leave it as-is if both helpers are needed in different places, but it does look strange.